kasa/tests/test_discovery.py

```diff
@@ -114,7 +114,7 @@ def mock_discover(self):
     "result": {
         "device_id": "xx",
         "owner": "xx",
-        "device_type": "SMART.TAPOPLUG",
+        "device_type": "SMART.TAPOBULB",
```

Comment:
Please use a name that is not going to be supported in the future either, this would fail already as soon as the tapobulb PR gets done :-)